include/morphio/shared_utils.tpp

```diff
@@ -1,8 +1,7 @@
-#include <cmath>
-
 #include <morphio/errorMessages.h>
 #include <morphio/types.h>
 
+# define PI           3.14159265358979323846f
```

Comment:
https://stackoverflow.com/questions/6563810/m-pi-works-with-math-h-but-not-with-cmath-in-visual-studio
also, don't use defines for constatnts, pls